spec/models/book_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Book, :type => :model do
```

Comment:
I love the new Ruby 1.9 hash syntax even it looks wired when both key and value are symbols.